parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -263,6 +270,7 @@ if (-not (Test-Path "C:\AzureData\windows\azurecnifunc.ps1")) {
 . c:\AzureData\windows\kubernetesfunc.ps1
 . c:\AzureData\windows\nvidiagpudriverfunc.ps1
 . c:\AzureData\windows\securetlsbootstrapfunc.ps1
+. c:\AzureData\windows\windowsciliumnetworkingfunc.ps1
```

Comment:
There is a chance of mismatched versions of this CSE and the windows scripts that means your ps1 won't exist on older VHDs. Probalby need code to protect against that case.

Reply:
Now checking that the script exists before dot-sourcing it.